### PR TITLE
Create new Harmony library

### PR DESCRIPTION
**Pull request by jfilowk**
**[CAN BE MERGED]**

Includes old Common, Repository, Futures, Testing

### CODE REVIEW

**Review by vilanovi — APPROVED** [review]

**Review by franmontiel — APPROVED** [review]
I just did a superficial check, I don't have enough background on Harmony Swift to have any significant comment on this

File: Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift [hunk 1]
```diff
@@ -14,13 +14,8 @@
...
 //
 
-#if DEBUG
-
 import Foundation
 
-/* These functions are internal to only be retrieved using '@testable import Harmony'. */
-
-func anyInt(minValue: Int = -32000, maxValue: Int = 32000) -> Int {
+public func anyInt(minValue: Int = -32000, maxValue: Int = 32000) -> Int {
```
Comment (AresProductions) [inline Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift:19]: Should all the test classes be public now? The idea was to have them internal so they can only be used on the test classes with import @testable.

Comment (AresProductions) [reply Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift:19]: Nevermind, it's on Harmony Testing now. :)

**Review by AresProductions — APPROVED** [review]

File: Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift (hunk 1 above)
Comment (vilanovi) [reply Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift:19]: exactly, we moved it to HarmonyTesting

**Comment by vilanovi** [comment]
we have added support for macOS as well in Carthage, which mas missing.

**Pull request merged**